chat-sdk-core/src/main/java/co/chatsdk/core/session/LocationManager.java

```diff
@@ -0,0 +1,20 @@
+package co.chatsdk.core.session;
+
+import co.chatsdk.core.base.BaseLocationAdapter;
+
+/**
+ * Created by Pepe on 01/25/19.
+ */
+
+public class LocationManager {
+
+    private final static LocationManager instance = new LocationManager();
+    public BaseLocationAdapter a;
+
+    protected LocationManager() {}
+
+    public static LocationManager shared() {
+        return instance;
+    }
+
+}
```

Comment:
Remove this class because it's not needed - since the location provider will be on the ChatSDK class.

Reply:
Was this class deleted? I can't see from the commit?

Reply:
Yes, I removed it